docs/glossary.md

```diff
@@ -24,3 +24,27 @@ manageability created need for BMC (Board Management
 Controllers), EC (Environmental Controllers) and even more
 specialized one like USB Power Delivery firmware. What may
 make that firmware also covered by above definition.
+
+## Dasharo Hardware Compatibility List Report
+
+Dasharo HCL Report dumps most important information about platform and backup
+SPI NOR flash.  Gathered information can be used for future analysis, debugging
+and recovery.  Optionally scripts upload dump to [Dasharo HCL Backup
+Server](https://backup.dasharo.com), so Dasharo Team improve open source
```

Comment:
```suggestion
Server](https://backup.dasharo.com), so Dasharo Team can improve open source
```

Reply:
Shouldn't we indicate that the backup.dasharo.com is not up yet (or is it)?

Reply:
@macpijan yes we should, maybe I will just add planned backup server.